src/main/java/org/quantumbadger/redreader/reddit/prepared/RedditRenderableCommentListItem.java

```diff
@@ -3,18 +3,19 @@
 import android.content.Context;
 import android.support.v7.app.AppCompatActivity;
 import android.view.View;
+
 import org.quantumbadger.redreader.common.RRThemeAttributes;
 
 public interface RedditRenderableCommentListItem {
 
 	CharSequence getHeader(
-			final RRThemeAttributes theme,
```

Comment:
The indentation here was correct before :) For line continuations it should be 2 tabs.